packages/docs/docs/lambda/serverlesslambdasetup.md

```diff
@@ -0,0 +1,203 @@
+---
+image: /generated/articles-docs-lambda-serverlesslambdasetup.png
+id: serverlesslambdasetup
+title: Serverless lambda setup
+slug: /lambda/serverless-lambdas-setup
+crumb: "Lambda"
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This is a follow up write up for a serverless function from [lamdbda permissions](/docs/lambda/lambda-role-permissions).
```

Comment:
```suggestion
This is a follow up write up for a serverless function from [Lambda permissions](/docs/lambda/lambda-role-permissions).
```

Reply:
I've changed it to 
This is a follow up write up for the serverless function from [using lambda without IAM user](/docs/lambda/using-lambda-without-iam-user).